gce_ovf_import_tests/test_suites/ovf_import_test_suite.go

```diff
@@ -71,10 +78,27 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	suffix := utils.RandString(5)
 	logger.Printf("Running TestSuite %q", testSuite.Name)
 
-	startupScript, err := ioutil.ReadFile("gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	startupScriptUbuntu3disks, err := ioutil.ReadFile(
+		"gce_ovf_import_tests/scripts/ovf_import_test_ubuntu_3_disks.sh")
+	if err != nil {
+		os.Exit(1)
+	}
+	startupScriptLinuxSingleDisk, err := ioutil.ReadFile(
+		"daisy_integration_tests/scripts/post_translate_test.sh")
+	if err != nil {
+		os.Exit(1)
+	}
+	startupScriptWindowsSingleDisk, err := ioutil.ReadFile(
+		"daisy_integration_tests/scripts/post_translate_test.ps1")
+	if err != nil {
+		os.Exit(1)
+	}
+	startupScriptWindowsTwoDisks, err := ioutil.ReadFile(
+		"gce_ovf_import_tests/scripts/ovf_import_test_windows_2k12_r2_two_disks.ps1")
```

Comment:
May I have your help to understand why ovf_import_test_windows_2k12_r2_two_disks.ps1 is under different folder from  post_translate_test.ps1? And why it's specifically for 2k12? (I read the file but don't see an obvious signal why it's only for 2k12)

Reply:
2k12_r2 OVA has two disks and needs additional code to check that. 2k16 has just one disk so we can use standard post translate test to validate it.

Reply:
I understand this one has two disks and need additional code for that. My question is whether it also works for 2016 and other windows versions. We may have the requirement to produce more e2e tests, and may reuse this script for different versions, if it works.

Reply:
It should work, though I don't see us adding more windows 2012/2016 tests with two disks specifically.

Reply:
Then I would suggest to change the file name to "ovf_import_test_windows_two_disks" to make it accurate. It's your call

Reply:
Done